api/enums.py

```diff
@@ -53,6 +53,11 @@ class EdgeTtsVoiceGender(Enum):
     FEMALE = "Female"
 
 
+class XVASynthModel(StrEnum):
```

Comment:
No need for StrEnum here I think - which might cause problems with pydantic serialization

Reply:
Removed enums